catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -160,7 +182,9 @@ private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable>
                 operationPolicy.putAll(policy);
             }
         }
-        return new PolicyResponseImpl(operationPolicy, operationPolicy);
+        itemPolicy.putAll(operationPolicy);
```

Comment:
Previously the operation policy was used for both, so I thought we would keep those policy settings as well as the ones we define ourselves.